.mailmap

```diff
@@ -185,6 +185,8 @@
 #
 # See also: 'git shortlog --help' and 'git check-mailmap --help'.
 #
+
+
```

Comment:
Why do we need these lines here ?

Reply:
Yes, this is the unnecessary line i will fix it!